lib/moku/pipeline/exec.rb

```diff
@@ -17,23 +17,16 @@ def self.handles?(command)
       end
 
       def call
-        step :retrieve_signature
-        step :construct_release
         step :run_command
```

Comment:
I'm not familiar with `step`, but it looks like this is queuing calls to private methods? So, previously, `call` would queue `command.signature`, the construction of `@release` based on that signature, and then `run_command`, and now it's letting go of `@release` and `@signature` in favor of having `release` just be a method based not only on `command.signature.deploy` but also on `command.release_id`.

And then I guess `command` is an instance of `Moku::Command::Exec`, and it's using the new `release_id` method you added to that?

And then `Release` is `Moku::Release` (unrelated to `Moku::Pipeline::Releases`), which, if not given an explicit `release_dir` will set the directory to match the timestamp of the release's creation (not the release we want, but each command also has a corresponding `release` object with its own id), so now that directory is set.

I think this way of getting rid of `@signature` and `@release` is a definitely good design decision. This version of `Exec` is much cleaner. It's nice that this doesn't involve adding any code to `Release`, since it's already ready to accept a `release_dir` argument. I think any difficulty I had in understanding this was more due to my unfamiliarity than anything else. These changes make sense.